arwen/contexts/runtime.go

```diff
@@ -40,23 +41,27 @@ type runtimeContext struct {
 	validator       *wasmValidator
 	instanceBuilder arwen.InstanceBuilder
 	errors          arwen.WrappableError
+
+	flagFixOOGReturnCode *atomic.Flag
 }
 
 // NewRuntimeContext creates a new runtimeContext
 func NewRuntimeContext(
 	host arwen.VMHost,
 	vmType []byte,
 	builtInFuncContainer vmcommon.BuiltInFunctionContainer,
+	flagFixOOGReturnCode *atomic.Flag,
```

Comment:
no. please do not give pointer of the flag.

Reply:
make a pulbic function on arwen main context which returns if this flag is true or false.

Reply:
Replaced with getter on `vmHost`.